src/coreclr/nativeaot/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -3,14 +3,12 @@
   <!-- Note that various places in SPCL depend on this resource name i.e. TplEventSource -->
   <ItemGroup Label="Embedded Resources">
     <EmbeddedResource Include="src\Resources\Strings.resx">
+      <LogicalName>System.Private.CoreLib.Strings.resources</LogicalName>
```

Comment:
The default logical name included `.src.` for some reason (it basically looked like a relative path to the csproj). I couldn't reverse engineer why.

The libraries partition has a bunch of convention-based rules in `eng/resources.targets` that we don't include because `src/coreclr` doesn't follow the convention. eng/illink.targets basically expects that and Mono's and CoreCLR's CoreLib.csproj files were basically hacked to make it work without including `resources.targets`.

Reply:
Does the same problem exists for all other `src/coreclr/nativeaot/System.*` assemblies?

Reply:
Ok, I guess the other assemblies have slightly different directory structure that makes it work.

Reply:
OK, I think I managed to make the current state an improvement instead of adding a hack.